src/cmd-compress

```diff
@@ -43,10 +43,12 @@ os.mkdir(tmpdir)
 
 at_least_one = False
 imgs_to_compress = []
+imgs_to_skip = ["iso", "vmware"]
+
 for img_format in buildmeta['images']:
 
     # Don't try to compress ISO images
-    if img_format == 'iso':
+    if img_format in imgs_to_skip:
```

Comment:
Do you think printing an informational message about the skipped artifact is worthwhile?

Reply:
Looks like this was fixed too, merging!

Reply:
Heh, actually I was asking if we *should* print a message (the last revision had no message).  NBD.